src/web/app/src/components/Posts/Post.tsx

```diff
@@ -360,9 +415,17 @@ const PostComponent = ({ postUrl, currentPost, totalPosts }: Props) => {
 
   // Listen for click events
   useEffect(() => {
-    window.document.addEventListener('click', handleZoom);
+    window.document.addEventListener('click', handleClick);
```

Comment:
I think you need to move `handleClick` into the closure:

```js
useEffect(() => {
  const handleClick = ...;
  window.document.addEventListener('click', handleClick);
  return () => {
    window.document.removeEventListener('click', handleClick); 
  }; 
```

Question: are you doing this at the right level?  Won't this mean we'll have one click event handler on the document for every post vs. one for all posts?  I think this needs to go higher up in the component stack, and you pass make this a controlled component.

Reply:
Confirm that I'm right about this.  cc @DukeManh

Reply:
`handleClick` refers to the same function so I doubt I will make another listener.
I agree that it should be moved up higher in the tree, but the posts content loads individually so I don't know if the listener will apply for the posts that load after